lib/src/views/Clock/Clock.tsx

```diff
@@ -167,8 +60,163 @@ export const styles = (theme: Theme) =>
       left: '50%',
       transform: 'translate(-50%, -50%)',
     },
-  });
+    amButton: {
+      zIndex: 24,
```

Comment:
Does 24 answer to a specific logic? Do you need the property in the first place?

Reply:
Thanks its debugging stuff